src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -307,7 +306,6 @@ private function getTreeForUuid(Request $request, $uuid)
                     $webspace,
                     $language,
                     $excludeGhosts,
-                    $excludeShadows,
```

Comment:
Are you sure that's the right way to fix this? From what I see the `getNodesTree` method has the exact parameters being passed here. This way the `appendWebspaceNode` parameter will be interepreted as `excludeShadows` in the method.